README.md

```diff
@@ -1,2 +1,1388 @@
+such as security, data monitoring and reporting.
+
+For example, `Stripe` is like having a multiple
+**payment processors**, **payment gateways** and **merchant account**
+bundled into one, 
+along with a [myriad of other features](https://stripe.com/en-pt).
+
+## `Stripe` 🚀
+
+`Stripe` is considered by many to be 
+the [*de facto*](https://trends.builtwith.com/payment/Stripe)
+way of accepting credit cards
+and electronic payments on the web. 
+Beyond collecting card payments 
+it has a number of additional features, 
+including: 
+[smart retries](https://stripe.com/docs/billing/revenue-recovery/smart-retries),
+[automatic card updater](https://stripe.com/docs/saving-cards),
+[fraud tooling](https://stripe.com/en-gb-pt/radar), 
+and other 
+[add-ons](https://stripe.com/partners/directory). 
+
+Until starting to research this in-depth,
+we were considering using `Stripe`
+because we've used in previous projects.
+But then we discovered `Paddle`!
+
+## `Paddle` 😮
+
+`Paddle` is a new class of payment processor
+that includes all additional services in their simple fee structure.
+Their slogan is "_The better way to sell software_".
+Which immediately got our attention as that is what we are selling! 
+
+While `Stripe` can be compared to a payment gateway
+that deals with multiple channels, 
+`Paddle` offers similar features
+but acts a *reseller service* - 
+**Merchant of Record (MoR)**.
+
+A MoR is a term to describe the legal entity
+selling goods or services to an end customer.
+~ [paddle.com/blog/what-is-merchant-of-record](https://www.paddle.com/blog/what-is-merchant-of-record)
+It's who the end customer owes payment for their purchase,
+and it is who handles payments and liability for each transaction.
+This is great for *tax handling*, 
+[which is especially relevant in Europe](https://www.outseta.com/posts/startup-payment-processing)
+and one of the reasons people 
+[consider `Paddle` in lieu of `Stripe`](https://splitbee.io/blog/why-we-moved-from-stripe-to-paddle).
+`Stripe` is making strides in also having
+[better tax compliance](https://stripe.com/newsroom/news/taxjar)
+but it's not quite there, at the moment of writing.
+
+![mor](https://user-images.githubusercontent.com/194400/210158068-92f2ec93-c108-4789-bc03-fd9b34c472bf.png)
+
+So businesses can choose to be their own merchant of record
+and setup an infrastructure and processes needed to manage
+payments with `Stripe` and deal with liabilities
+and tax handling themselves.
+*Or* they can use MoR service providers like `Paddle`
+who take the burden of all of payment processing 
+and legal compliance away. Of course, these usually incur higher fees than `Stripe`.
+
+## I'm confused. Which one should I choose? 🤷‍♀️
+
+[That's a great question](https://www.indiehackers.com/post/stripe-vs-paddle-89161b0d5c).
+
+[Which seems to asked](https://splitbee.io/blog/why-we-moved-from-stripe-to-paddle)
+[over](https://stackshare.io/stackups/paddle-vs-stripe),
```

Comment:
Please don't do this. `[over] and [over] and [over]` is not good link text. https://github.com/dwyl/contributing/issues/119
I _understand_ that you are emphasising the fact that _many_ people have had these same questions.
But as a document that we want to read, maintain and have others _contribute_ to. 
We want to set high standards for link text. 
Please/thanks! 🙏